python/pyproject.toml

```diff
@@ -36,7 +36,7 @@ requests-toolbelt = "^1.0.0"
 langsmith-pyo3 = { version = "^0.1.0rc2", optional = true }
 zstandard = "^0.23.0"
 rich = {version = "^13.9.4", optional = true}
-pytest = {version = ">=7.0.0", optional = true}
+pytest = {version = ">=8.0.0", optional = true}
```

Comment:
is this necessary? does it not work with v7 after the fix?

Reply:
This is to fix the VCR issue from Friday

Reply:
Never mind, I'm dumb